oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticDocument.java

```diff
@@ -81,6 +91,11 @@ void addProperty(String fieldName, Object value) {
         properties.put(fieldName, value);
     }
 
+    void addSimField(String name, Blob value) throws IOException{
```

Comment:
Could you rename the method to "addSimilarityField"? I first thought it means "addSimulatedField" :-)